jquery.flot.navigate.js

```diff
@@ -522,9 +545,19 @@ Licensed under the MIT License ~ http://threedubmedia.googlecode.com/files/MIT-L
             if (isNaN(delta.x)) delta.x = 0;
             if (isNaN(delta.y)) delta.y = 0;
 
-            var axes = plot.getAxes();
+            var axes;
+
+            if (panAxes) {
+                axes = panAxes;
+            } else {
+                axes = plot.getAxes();
+            }
```

Comment:
1. pan, smartPan, and zoom should not share state (panAxes) with the event handlers. The event handlers should pass as arguments everything those 3 need. Consider those 3 functions part of a different file.
2. As much as possible you should stick to a single convention for the axes and panAxes: arrays or objects.